spec/models/budget/ballot/line_spec.rb

```diff
@@ -24,6 +24,13 @@
         expect(ballot_line).not_to be_valid
       end
 
+      it "is valid if insufficient funds, is approval voting, limits turned off" do
+        budget.update_attribute :voting_style, "approval"
+        group.update_attribute :voting_cant_exceed_heading_budget, false
+        investment.update_attribute :price, investment.price + 1
```

Comment:
Shouldn't this be `heading.price + 1` instead of `investment.price + 1`? 🤔

Reply:
Good question. Checking.